app/routes/circulars._archive._index.tsx

```diff
@@ -291,7 +340,152 @@ export default function () {
           </Button>
         </Form>
         {featureCircularsFilterByDate && (
-          <DateSelectorButton startDate={startDate} endDate={endDate} />
+          <>
+            <DateSelectorButton
+              startDate={startDate}
+              endDate={endDate}
+              onClick={() => {
+                setShowContent((shown) => !shown)
+                setShowDateRange(false)
+              }}
+              expanded={showContent}
+            />
+            {showContent && (
+              <DetailsDropdownContent className="margin-top-1">
+                <div className="maxw-card-xlg">
+                  <div className="display-flex flex-row">
+                    <div className="display-flex flex-column flex-align-start margin-1">
+                      <Radio
+                        id="radio-alltime"
+                        name="radio-date"
+                        value="undefined"
+                        defaultChecked={true}
+                        label="All Time"
+                        onChange={(e) => {
+                          setFuzzyTime(e.target.value)
+                        }}
+                      />
+                      <Radio
+                        id="radio-hour"
+                        name="radio-date"
+                        value="hour"
+                        label="Last hour"
+                        onChange={(e) => {
+                          setFuzzyTime(e.target.value)
+                        }}
+                      />
+                      <Radio
+                        id="radio-today"
+                        name="radio-date"
+                        value="today"
+                        label="Today"
+                        onChange={(e) => {
+                          setFuzzyTime(e.target.value)
+                        }}
+                      />
```

Comment:
To reduce duplication of code, please generate all of these radio buttons for the fuzzy date ranges by looping over `dateSelectorLabels`:

```jsx
{Object.entries(dateSelectorLabels).map(([key, label]) => <Radio ... />)}

Reply:
> To reduce duplication of code, please generate all of these radio buttons for the fuzzy date ranges by looping over `dateSelectorLabels`:
> 
> ```js
> {Object.entries(dateSelectorLabels).map(([key, label]) => <Radio ... />)}
> ```

would I break dateSelectorLabels into sublists to account for them needing to be in different columns? I may be misunderstanding how the Grid object works though

Reply:
switched to using the Grid structure in 8c3016d, this is the result:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/57914086/f069b22f-c39f-4154-a8bf-dcf0e8030fc7)

It actually fixes the issue I had described regarding the horizontal alignment of the radio buttons when on mobile, which is nice:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/57914086/9bbdbebb-aca8-4006-9dc4-1571154a02ac)

Reply:
> would I break dateSelectorLabels into sublists to account for them needing to be in different columns?

No, it will wrap automatically.

Reply:
> To reduce duplication of code, please generate all of these radio buttons for the fuzzy date ranges by looping over `dateSelectorLabels`:
> 
> ```js
> {Object.entries(dateSelectorLabels).map(([key, label]) => <Radio ... />)}
> ```

Should be added as of 88b8669